test/AbstractionMapperTests.cs

```diff
@@ -0,0 +1,309 @@
+using FluentAssertions;
+using Microsoft.Extensions.AI;
+using Moq;
+using NUnit.Framework;
+using OllamaSharp;
+using OllamaSharp.MicrosoftAi;
+using OllamaSharp.Models.Chat;
+
+namespace Tests;
+
+public class AbstractionMapperTests
```

Comment:
Some tests are commented out or even missing to cover the AbstractionMapper. This is a follow-up to the missing information about how to map certain properties. See my prior comment for the AbstractionMapper.